test/test_nn.py

```diff
@@ -77,7 +77,8 @@
 
 dtype2prec = {torch.float: 1e-5,
               torch.double: 1e-5,
-              torch.half: 1e-2}
+              torch.half: 1e-2,
+              torch.bfloat16: 1e-1}
```

Comment:
Is there any motivation for using an even higher threshold than fp16?

Reply:
Mainly from experimental results. Tests were failing with `1e-2` and `1e-1` has been used for blas tests as well(even for cpu in some cases). Also fp16 has better precision than bfloat16 theoretically?